src/snowcli/cli/stage/manager.py

```diff
@@ -60,15 +77,16 @@ def _put(
             stage_path = self.get_standard_stage_name(stage_path)
             local_resolved_path = path_resolver(str(local_path))
             cursor = self._execute_query(
-                f"put file://{local_resolved_path} {stage_path} "
+                f"put file://{local_resolved_path} {self.quote_stage_name(stage_path)} "
                 f"auto_compress=false parallel={parallel} overwrite={overwrite}"
             )
         return cursor
 
     def remove(self, stage_name: str, path: str) -> SnowflakeCursor:
         stage_name = self.get_standard_stage_name(stage_name)
         path = path if path.startswith("/") else "/" + path
-        return self._execute_query(f"remove {stage_name}{path}")
+        to_remove = self.quote_stage_name(f"{stage_name}{path}")
```

Comment:
What do you think about `quoted_stage_name`?

Reply:
Sure, done